tools/sklearn/test-data/svc_prediction_result03.tabular

```diff
@@ -33,9 +33,9 @@
 2	-99	26	-108	1
 2	-81	19	-110	0
 2	-108	21	-108	1
-2	-92	27	-106	0
+2	-92	27	-106	1
 2	-88	2	-106	3
```

Comment:
Have you noticed that the prediction results change back and forth?

Reply:
@qiagu I think we are predicting the correct class for the sample shown in ```line 36 ```
https://github.com/anuprulez/galaxytools/blob/sklearn_h5_model/tools/sklearn/test-data/train_set.tabular#L36

https://github.com/anuprulez/galaxytools/blob/sklearn_h5_model/tools/sklearn/test-data/test_set.tabular#L36